src/runtime/scheduler_gba.S

```diff
@@ -0,0 +1,23 @@
+.section .text.tinygo_scanCurrentStack
+.global  tinygo_scanCurrentStack
+.type    tinygo_scanCurrentStack, %function
+tinygo_scanCurrentStack:
+    // Save callee-saved registers onto the stack.
+    #if defined(__thumb2__)
+    push {r4-r11, lr}
```

Comment:
The GBA doesn't support Thumb 2.
(In ARM mode, however, you could do the push as above).

Reply:
Fixed.